### PR TITLE
Solved issue #86 - CSS file in <script> tag

### PR DESCRIPTION
**Pull request by devwizard4047**
Removed CSS link from head.html
Preview link - https://devwizard4047.github.io/2012.fossasia.org/

### CODE REVIEW

File: CNAME [hunk 1]
```diff
@@ -1 +0,0 @@
-2012.fossasia.org
```
Comment (akshatnitd) [inline CNAME]: Why did you remove the CNAME file ?

**Review by akshatnitd — CHANGES_REQUESTED** [review]

File: CNAME (hunk 1 above)
Comment (devwizard4047) [reply CNAME]: It was creating some issue with rendering on GitHub pages, not really sure what.

Comment (akshatnitd) [reply CNAME]: That might be because of some other issues

Comment (akshatnitd) [reply CNAME]: Add back the file

**Comment by devwizard4047** [comment]
I have added CNAME again. Please check.
Preview link - https://devwizard4047.github.io/2012.fossasia.org/

**Review by akshatnitd — COMMENTED** [review]
Just need to squash the commits and it will be ready to merge :)
